packages/mobile/src/types/gesture.ts

```diff
@@ -1,3 +1,4 @@
 import { GestureResponderEvent } from 'react-native'
 
-export type GestureResponderHandler = (e: GestureResponderEvent) => void
+export type GestureResponderHandler = ((e: GestureResponderEvent) => void) &
```

Comment:
would changing to e? solve the problem?

Reply:
Maybe, good idea! It was `TouchableOpacity` that was expecting this type